src/include/OSL/oslconfig.h

```diff
@@ -152,6 +152,25 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     #endif
 #endif // OSL_DEBUG
 
+#if defined(_MSC_VER)
+    #define OSL_PRAGMA(aUnQuotedPragma) __pragma(aUnQuotedPragma)
+#else
+    #define OSL_PRAGMA(aUnQuotedPragma) _Pragma(#aUnQuotedPragma)
```

Comment:
Why does this work? Oh, `_Pragma` is C99. Does that work for all non-MSVS C++ compilers we might deal with? (I guess it does if it passed the CI.)

Reply:
https://en.cppreference.com/w/cpp/preprocessor/impl
_Pragma is part of c++11 standard.  
It was not implemented in the versions of MSVC we were using.
https://developercommunity.visualstudio.com/content/problem/325605/vc-does-not-support-the-standard-pragma.html
So we have this conditional compilation to use the existing MSVC __pragma which doesn't accept quotes.  So we standardized OSL_PRAGMA to be unquoted and supply the required quotes for _Pragma itself.